models/store/src/store/generator.ts

```diff
@@ -26,7 +26,7 @@ const generator = Generator<TStore>({
     key: fake((f) => f.lorem.slug()),
     name: fake(() => LocalizedString.random()),
     languages: [oneOf('en-US', 'de-DE', 'es-ES')],
-    countries: [fake((f) => f.location.countryCode())],
+    countries: [fake((f) => ({ code: f.location.countryCode() }))],
```

Comment:
The actual shape of this property is not an array of strings but an array of objects containing a `code` property with the ISO country code.

As a follow up, it would be great if we could make typescript to warn us when we try to provide an invalid value to a property of the model at development time.
We already have [this proposal](https://commercetools.atlassian.net/browse/FEC-62) for the fe chapter.